src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/FormsToolkit.ios.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UIKit;
+
+namespace Xamarin.CommunityToolkit.Helpers
+{
+	public static class FormsToolkit
+	{
+		static bool? isiOS9OrNewer;
+
+		public static bool IsiOS9OrNewer
```

Comment:
Ah, I notice we have this in Xamarin.Forms but everything is internal.

Reply:
When this PR was originally written for XF it was dependent on a lot of internal bits of XF. I am not a big fan of having a `FormsToolkit` object.

What is the official min of iOS that Xamarin.Forms supports? I just took a glance at the MAUI docs and it says XF support is 9+ and MAUI is 10+. If we want to take the hard stance of XCT supports 9+ we can remove this entire class.

Reply:
I took a look at the official Xamarin.Forms documentation and found this article that states iOS must be 9 or greater. Maybe we should just pull this code out and make the assumption since anything less is not supported.
* https://docs.microsoft.com/en-us/xamarin/get-started/supported-platforms?tabs=windows

![image](https://user-images.githubusercontent.com/17751436/101918908-f4514780-3b97-11eb-9114-86750f65f929.png)

Reply:
code push on the way with this fixed